aws-rds-customdbengineversion/src/main/java/software/amazon/rds/customdbengineversion/BaseHandlerStd.java

```diff
@@ -83,6 +83,7 @@ public abstract class BaseHandlerStd extends BaseHandler<CallbackContext> {
     private final FilteredJsonPrinter EMPTY_FILTER = new FilteredJsonPrinter();
 
     protected final HandlerConfig config;
+    protected RequestLogger requestLogger;
```

Comment:
It looks like you missed initializing the RequestLogger for this resource. Make sure you initialize `this.requestLogger = requestLogger`.

Reply:
Resolved in the latest commit.